tools/cli/NearObjectCli.cxx

```diff
@@ -269,6 +285,47 @@ NearObjectCli::AddSubcommandUwbRangeStart(CLI::App* parent)
     rangeStartApp->add_option("--ResultReportConfiguration", uwbConfig.resultReportConfigurationString)->capture_default_str();
 
     rangeStartApp->parse_complete_callback([this] {
+        // Validate NumberOfControlees
+        if (m_cliData->uwbConfiguration.multiNodeMode == uwb::protocol::fira::MultiNodeMode::Unicast) {
+            if (m_cliData->uwbConfiguration.numberOfControlees != 1) {
+                std::cerr << "Only 1 controlee expected in Unicast mode" << std::endl;
+            }
+        } else {
+            if (m_cliData->uwbConfiguration.numberOfControlees < 1 || m_cliData->uwbConfiguration.numberOfControlees > 8) {
+                std::cerr << "Invalid number of controlees. Must be 1 <= N <= 8" << std::endl;
+            }
+        }
+        // Set MAC addresses
+        std::optional<uwb::UwbMacAddress> controllerMacAddress;
+        std::optional<uwb::UwbMacAddress> controleeMacAddress;
+
+        const auto macAddressType = m_cliData->uwbConfiguration.macAddressMode == uwb::UwbMacAddressType::Extended ? uwb::UwbMacAddressType::Extended : uwb::UwbMacAddressType::Short;
+
+        // TODO: Support multiple controlees
+        if (m_cliData->uwbConfiguration.deviceType == uwb::protocol::fira::DeviceType::Controller) {
+            controllerMacAddress = uwb::UwbMacAddress::FromString(m_cliData->deviceMacAddress, macAddressType);
+            controleeMacAddress = uwb::UwbMacAddress::FromString(m_cliData->destinationMacAddress, macAddressType);
+        } else { // uwb::protocol::fira::DeviceType::Controlee
+            controllerMacAddress = uwb::UwbMacAddress::FromString(m_cliData->destinationMacAddress, macAddressType);
+            controleeMacAddress = uwb::UwbMacAddress::FromString(m_cliData->deviceMacAddress, macAddressType);
+        }
+
+        if (!controllerMacAddress.has_value()) {
+            std::cerr << "Invalid ControllerMacAddress" << std::endl;
+        } else {
+            m_cliData->uwbConfiguration.controllerMacAddress = controllerMacAddress.value();
+            // TEST
```

Comment:
Does this comment and/or associated code need to be removed, or are they here intentionally?